searxng/searxng
# [mod] simple-theme: on small devices show category icon, not name

## What does this PR do?

[mod] simple-theme: on small devices show category icon, not name

**before**

![grafik](https://user-images.githubusercontent.com/554536/144761115-5bde1005-b91d-4fb6-9b86-48614f1af1f0.png)

**after**

![grafik](https://user-images.githubusercontent.com/554536/146212267-858d7df0-883d-4f82-bd24-d4b74a49acf4.png)

## How to test this PR locally?

To test I patched settings.yml

1. add a new category "TV" to see how *new* categories will be rendered (the icon)
2. activate autocomplete, to see autocompletion is positioned well
3. add `Content-Security-Policy` to check CSP

```diff
diff --git a/searx/settings.yml b/searx/settings.yml
index 305b254c..0ec979cd 100644
--- a/searx/settings.yml
+++ b/searx/settings.yml
@@ -16,7 +16,7 @@ search:
   # Existing autocomplete backends: "dbpedia", "duckduckgo", "google",
   # "startpage", "swisscows", "qwant", "wikipedia" - leave blank to turn it off
   # by default.
-  autocomplete: ""
+  autocomplete: "google"
   # Default search language - leave blank to detect from browser information or
   # use codes from 'languages.py'
   default_lang: ""
@@ -48,7 +48,7 @@ server:
 
   secret_key: "ultrasecretkey"  # change this!
   # Proxying image results through searx
-  image_proxy: false
+  image_proxy: true
   # 1.0 and 1.1 are supported
   http_protocol_version: "1.0"
   # POST queries are more secure as they don't show up in history but may cause
@@ -60,6 +60,7 @@ server:
     X-Download-Options: noopen
     X-Robots-Tag: noindex, nofollow
     Referrer-Policy: no-referrer
+    Content-Security-Policy: "default-src 'self'; img-src 'self' data: ;"
 
 ui:
   # Custom static path - leave it blank if you didn't change
@@ -70,7 +71,7 @@ ui:
   # it decreases the privacy, since the browser can records the page titles.
   query_in_title: false
   # ui theme
-  default_theme: oscar
+  default_theme: simple
   # Default interface locale - leave blank to detect from browser information or
   # use codes from the 'locales' config section
   default_locale: ""
@@ -1604,6 +1605,7 @@ engines:
   - name: mediathekviewweb
     engine: mediathekviewweb
     shortcut: mvw
+    categories: TV
 
   # - name: yacy
   #   engine: yacy
```
